CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 # v0.55.0
 
 ## 🛑 Breaking changes 🛑
+- Add `linux-ppc64le` architecture to cross build tests in CI
```

Comment:
The changelog in this repo works differently, check the contribution guidelines. In short, add a file to ./unreleased. That said, I don't think we should have this listed in the changelog.

Reply:
@adilhusain-s, here are the instructions for adding a changelog entry: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-a-changelog-entry

Please remove this line and add an entry according to these instructions.